code/modules/mob/say.dm

```diff
@@ -1,3 +1,22 @@
+var/global/list/special_channel_keys = list(
+	"r" = MESSAGE_MODE_RIGHT,
+	"l" = MESSAGE_MODE_LEFT,
+	"i" = MESSAGE_MODE_INTERCOM,
+	"h" = MESSAGE_MODE_DEPARTMENT,
+	"+" = MESSAGE_MODE_SPECIAL,    //activate radio-specific special functions
+	"w" = MESSAGE_MODE_WHISPER,
+	"ê" = MESSAGE_MODE_RIGHT,
+	"ä" = MESSAGE_MODE_LEFT,
+	"ø" = MESSAGE_MODE_INTERCOM,
```

Comment:
are these supposed to be nordletters, or is that weird artifact when moving cyrillic keys?

Reply:
It was present in the original PR.